docs/source/en/using-diffusers/text-img2vid.md

```diff
@@ -10,44 +10,34 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Text or image-to-video
+# Video generation
```

Comment:
Should this guide also include the animation generation models we have (`AnimateDiff`, for example)?